index.js

```diff
@@ -44,6 +47,9 @@ module.exports = function(opts) {
       return callback();
     }
 
+    // If HTMLBars, output would be prefixed with Ember.HTMLBars.template function
+    compiled = isHTMLBars ? 'export default Ember.HTMLBars.template(' + compiled + ');' : compiled;
```

Comment:
It seems like a big assumption to make, "wrap everything for Ember and HTMLBars if they pass a compiler function." Is there a better way to detect if they intend to compile HTMLBars?